sdks/python/apache_beam/io/fileio.py

```diff
@@ -745,13 +745,13 @@ def _check_orphaned_files(self, writer_key):
       orphaned_files = [m.path for m in match_result[0].metadata_list]
 
       if len(orphaned_files) > 0:
-        _LOGGER.info(
```

Comment:
Sometimes (or most time) orphan files are harmless, for example, it is observed when write file bundle fails with intermittent error, and lwork item ater get retried. Need to rephrase this warning to say sometime it is harmless to avoid confusion.